rest-api/dao/participant_summary_dao.py

```diff
@@ -379,6 +380,34 @@ def to_client_json(self, model):
 
     return result
 
+  def to_client_csv(self, model):
+    from flask import Response
+    import StringIO
+    # from unicode_csv import UnicodeWriter
+
+
+    def iter_csv(data):
+      line = StringIO.StringIO()
+      writer = csv.writer(line)
```

Comment:
Another thing is: the default CSV libraries don't read or write Unicode data. We need to return Unicode properly... see:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/unicode_csv.py#L12

for how we're handling that in other contexts.